spec/controllers/pageflow/editor/entries_controller_spec.rb

```diff
@@ -53,6 +53,25 @@ module Pageflow
 
         expect(response.status).to eq(401)
       end
+
+      it 'renders entry type specific body fragment' do
```

Comment:
this test is for the `seed` fragment, not `body`

Reply:
All code changes approved, but you could check the wording in `doc/creating_entry_types.md` before merge.

Reply:
Ah yes, a few missing words, right? Or more general suggestions how it could be improved?

Reply:
Fixed the typos.

Reply:
Fixed the types.